generate-stackbrew-library.sh

```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 declare -A release_channel=(
-  [stable]='2020.09'
+  [stable]='2021.01'
```

Comment:
Can't we have this in a `stable.txt` file like the `latest` and `develop` in the same file? Or can't we just use the `latest`?